.travis.yml

```diff
@@ -11,3 +11,11 @@ install:
 
 script:
   - npm run lint
+
+deploy:
+  provider: npm
+  email: "ben@daostack.io"
+  api_key: $NPM_API_TOKEN
```

Comment:
need instructions on how to set this (`.env`? )

Reply:
No need, this is already in travis and should only be executed by it